src/index.ts

```diff
@@ -1,5 +1,13 @@
-export { default as EOSIOAuthProvider } from './authProvider/eosioAuthProvider'
-export { default as EthereumAuthProvider } from './authProvider/ethereumAuthProvider'
+// Re-exports from blockchain-utils-linking
+export {
+  CosmosAuthProvider,
+  EosioAuthProvider,
+  EthereumAuthProvider,
+  FilecoinAuthProvider,
+  PolkadotAuthProvider,
+} from '@ceramicnetwork/blockchain-utils-linking'
```

Comment:
Can't we just export all of the providers like this?
```suggestion
export * from '@ceramicnetwork/blockchain-utils-linking'
```
Then we wouldn't need to update 3id-connect when support for new blockchains are added in the util lib.

Reply:
I don't think it's a good practice to "re-export all", it could lead to issues with breaking changes in the upstream package causing breaking changes in minor/patch versions of 3ID Connect.